addons/website_slides/models/slide_slide.py

```diff
@@ -546,38 +546,40 @@ def _get_can_publish_error_message(self):
     # ORM Overrides
     # ---------------------------------------------------------
 
-    @api.model
-    def create(self, values):
-        # Do not publish slide if user has not publisher rights
-        channel = self.env['slide.channel'].browse(values['channel_id'])
-        if not channel.can_publish:
-            # 'website_published' is handled by mixin
-            values['date_published'] = False
-
-        if values.get('is_category'):
-            values['is_preview'] = True
-            values['is_published'] = True
-        if values.get('is_published') and not values.get('date_published'):
-            values['date_published'] = datetime.datetime.now()
-
-        slide = super(Slide, self).create(values)
-
-        # avoid fetching external metadata when installing the module (i.e. for demo data)
-        # we also support a context key if you don't want to fetch the metadata when creating a slide
-        if any(values.get(url_param) for url_param in ['url', 'video_url', 'document_google_url', 'image_google_url']) \
-           and not self.env.context.get('install_mode') \
-           and not self.env.context.get('website_slides_skip_fetch_metadata'):
-            slide_metadata, _error = slide._fetch_external_metadata()
-            if slide_metadata:
-                # only update keys that are not set in the incoming values
-                slide.update({key: value for key, value in slide_metadata.items() if key not in values.keys()})
-
-        if not 'completion_time' not in values:
-            slide._on_change_document_binary_content()
-
-        if slide.is_published and not slide.is_category:
-            slide._post_publication()
-        return slide
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            # Do not publish slide if user has not publisher rights
+            channel = self.env['slide.channel'].browse(vals['channel_id'])
+            if not channel.can_publish:
+                # 'website_published' is handled by mixin
+                vals['date_published'] = False
+
+            if vals.get('is_category'):
+                vals['is_preview'] = True
+                vals['is_published'] = True
+            if vals.get('is_published') and not vals.get('date_published'):
+                vals['date_published'] = datetime.datetime.now()
+
+        slides = super().create(vals_list)
+
+        for vals, slide in zip(vals_list, slides):
```

Comment:
You usually put the record as first argument in other zips (such reviewer much comment).